extensions/blocks/map/component.js

```diff
@@ -177,18 +177,30 @@ export class Map extends Component {
 	};
 	// Various map functions
 	sizeMap = () => {
+		const { mapHeight } = this.props;
 		const { map } = this.state;
 		const mapEl = this.mapRef.current;
-		const blockWidth = mapEl.offsetWidth;
-		const maxHeight =
-			window.location.search.indexOf( 'map-block-counter' ) > -1
-				? window.innerHeight
-				: window.innerHeight * 0.8;
-		const blockHeight = Math.min( blockWidth * ( 3 / 4 ), maxHeight );
-		mapEl.style.height = blockHeight + 'px';
+		if ( mapHeight ) {
+			mapEl.style.height = mapHeight + 'px';
```

Comment:
This seems odd for React; I'd expect to see `style={ { height: ... } }` in JSX instead.

Reply:
Oh I see it's what was already done before in the block: https://github.com/Automattic/jetpack/pull/14706/files#diff-fc2a014531c8c68c34dcc4c8292c5f21L188

Reply:
Yah, I left that as is, I assumed that the direct manipulation of `mapEl` is deliberate.